src/marqo/tensor_search/tensor_search.py

```diff
@@ -852,15 +894,35 @@ def _lexical_search(
             body["_source"] = dict()
         if body["_source"] is not False:
             body["_source"]["exclude"] = [f"*{TensorField.vector_prefix}*"]
+    
+    end_preprocess_time = timer()
+    total_preprocess_time = end_preprocess_time - start_preprocess_time
+    logger.info(f"search (lexical) pre-processing: took {(total_preprocess_time):.3f}s to process query.")
+    
+    # SEARCH TIMER-LOGGER (roundtrip)
+    start_search_http_time = timer()
     search_res = HttpRequests(config).get(path=f"{index_name}/_search", body=body)
 
+    end_search_http_time = timer()
+    total_search_http_time = end_search_http_time - start_search_http_time
+    num_results = len(search_res['hits']['hits'])
+    logger.info(f"search (lexical) roundtrip: took {(total_search_http_time):.3f}s to send search query (roundtrip) to Marqo-os and received {num_results} results.")
```

Comment:
Can we get the took time from the Marqo-os response too? In the same way we had "Marqo-os index" timing

Reply:
done